Docs/docs.js

```diff
@@ -0,0 +1,141 @@
+
+try{
+    //To toggle the main sidebar
+    const mainSidebar = document.querySelector("aside.tr-drawer-modal");
+    const hamburgers = document.querySelectorAll(".hamburger");
+    
+    hamburgers.forEach(btn=>btn.addEventListener('click', e => {
+        let toggleTarget = document.getElementById(btn.getAttribute('data-target'));
+        toggleTarget.classList.toggle('active');
+
+        if(toggleTarget.className.includes('tr-drawer-modal')){
+            toggleTarget.parentElement.classList.toggle('active') //To activate the BG overlay
+        }
+        e.stopPropagation();
+    })
+    )
+
+    mainSidebar.addEventListener('click',e=>{
+        e.stopPropagation(); // make sure the eventlistener on the body is not triggered when anything on the sidebar is clicked
+    });
+
+    document.body.addEventListener('click',(e)=>{
+        if(mainSidebar.className.includes('tr-drawer-modal')){
+            mainSidebar.classList.remove('active')
+            mainSidebar.parentElement.classList.remove('active');
+        }
+    })
+
+}
+catch(err){
+if(err.message.includes('Cannot read properties of null')) console.error(err);
+else throw err;
+}
```

Comment:
Code can be indented for better readability

Reply:
fixed in the latest commit

Reply:
Perfect